### PR TITLE
[FIX] Pass alpha on to glass brain when plotting connectome

### PR DESCRIPTION
**Pull request by koenhelwegen**
Minor fix: `alpha` in `plotting.plot_connectome` is currently not passed on to `plot_glass_brain()`.

### CODE REVIEW

**Review by ymzayek — COMMENTED** [review]
@koenhelwegen thanks for catching that! Can you update the PR title to have the prefix [FIX]? See [here](https://nilearn.github.io/stable/development.html#pr-structure). 

I think it might also be good to add a smoke test in `test_plot_connectome.py` with several different values for alpha. 

And then can you add an entry to `doc/changes/latest.rst` and your information to `doc/changes/names.rst`?

**Comment by koenhelwegen** [comment]
Thanks for quickly picking this up @ymzayek!

I added the smoke test, currently only with valid values for `alpha`. Do you think it is useful to test an error gets raised for invalid values as well?

File: doc/changes/latest.rst [hunk 1]
```diff
@@ -26,6 +26,7 @@ Fixes
 - Fix usage of ``scipy.stats.gamma.pdf`` in ``_gamma_difference_hrf`` function under ``nilearn/glm/first_level/hemodynamic_models.py``, which resulted in slight distortion of HRF (:gh:`3297` by `Kun CHEN`_).
 - Fix bug introduced due to a fix in the pre-release version of scipy (``1.9.0rc1``) which now enforces that elements of a band-pass filter must meet condition ``Wn[0] < Wn[1]``.
   Now if band-pass elements are equal :func:`~nilearn.signal.butterworth` returns an unfiltered signal with a warning (:gh:`3293` by `Yasmin Mzayek`_).
+- Fix bug in behavior of ``alpha`` in ``nilearn.plotting.plot_connectome`` (:gh:`3306` by `Koen Helwegen`_).
```
Comment (ymzayek) [inline doc/changes/latest.rst:29]:
```suggestion
- Fix bug in behavior of ``alpha`` in :func:`~plotting.plot_connectome` (:gh:`3306` by `Koen Helwegen`_).
```
The description can also be a bit more explicit, such as 

The parameter \`\`alpha\`\` is now correctly passed to :func:\`~plotting.plot_glass_brain\` in :func:\`~plotting.plot_connectome\`

**Comment by ymzayek** [comment]
> I added the smoke test, currently only with valid values for `alpha`. Do you think it is useful to test an error gets raised for invalid values as well?

Thanks! I don't think it would hurt but I'm not sure it is necessary since checking `alpha` comes from a matplotlib module and is not something implemented in nilearn.
Perhaps @NicolasGensollen or @jeromedockes can chime in on this.

File: doc/changes/latest.rst (hunk 1 above)
Comment (koenhelwegen) [reply doc/changes/latest.rst]: Thanks, updated in [e37b9ce](https://github.com/nilearn/nilearn/pull/3306/commits/e37b9cea58ec59ce3b7a36f29d6ffe56b37afd2f)

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks @koenhelwegen !

**Comment by NicolasGensollen** [comment]
@ymzayek  I don't think we need to test that an error is raised for invalid values of alpha because this is handled in matplotib AFAIK.

**Review by ymzayek — APPROVED** [review]
LGTM too!

**Comment by ymzayek** [comment]
Merging

**Pull request merged**